broker-cli/commands/wallet.js

```diff
@@ -232,24 +245,35 @@ async function networkStatus (args, opts, logger) {
   try {
     const client = new BrokerDaemonClient(rpcAddress)
     const { baseSymbolCapacities, counterSymbolCapacities } = await client.walletService.getTradingCapacities({market})
+    const { activeOutboundAmount: committedCounterSendCapacity, activeInboundAmount: committedBaseReceiveCapacity } = await client.orderService.getActiveFunds({ market, side: 'BID' })
```

Comment:
I think this should happen in the `getTradingCapacities` endpoint so that we can keep the CLI skinny.